### PR TITLE
podman-remote volume inspect|ls

### PR DESCRIPTION
**Pull request by baude**
add the ability to list and inspect volumes using the remote
client and varlink

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
the next volume based PR will enable tests and add API documentation to all volume related commands where applicable

**Comment by rhatdan** [comment]
LGTM
@jwhonce PTAL

File: cmd/podman/varlink/io.podman.varlink [hunk 1]
```diff
@@ -3,6 +3,33 @@
 interface io.podman
 
 
+# Version is the structure returned by GetVersion
```
Comment (mheon) [inline cmd/podman/varlink/io.podman.varlink]: Is this a mistake? Didn't this disappear in the earlier varlink PR?

File: libpod/runtime_volume.go [hunk 2]
```diff
@@ -154,3 +154,17 @@ func (r *Runtime) GetAllVolumes() ([]*Volume, error) {
 
 	return r.state.AllVolumes()
 }
+
+// GetVolumes accepts a string slice of volume names and returns a slice
+// of volumes
+func (r *Runtime) GetVolumes(volumes []string) ([]*Volume, error) {
```
Comment (mheon) [inline libpod/runtime_volume.go]: Call GetVolume in a loop

**Comment by baude** [comment]
fixed up per comments, ready for merge

File: cmd/podman/varlink/io.podman.varlink (hunk 1 above)
Comment (baude) [reply cmd/podman/varlink/io.podman.varlink]: fixed, likely due to rebases

**Pull request merged**